### PR TITLE
EP-276: CTA Clicked (log_in_submit)

### PR DESCRIPTION
**Pull request by sunday-okpoluaefe**
# 📲 What

Android: CTA Clicked (log_in_submit)

# 🤔 Why

Segment Implementation

# 🛠 How

- Created method ``` trackLogInButtonCtaClicked ``` in ``` AnalyticEvents ```

```
    /*
     * Sends data to the client when the login button is clicked.
     */
    fun trackLogInButtonCtaClicked() {
        val props: HashMap<String, Any> = hashMapOf(CONTEXT_PAGE.contextName to LOGIN.contextName)
        props[CONTEXT_CTA.contextName] = LOGIN_SUBMIT.contextName
        client.track(CTA_CLICKED.eventName, props)
    }
```

- Called method from ``` LoginViewModel ``` on ``` Log in ``` button clicked

``` 
   logInButtonClicked
       .compose(bindToLifecycle())
       .subscribe{ this.lake.trackLogInButtonCtaClicked() }
```

# 👀 See

Screenshot 2021-03-13 at 07 22 06

| Before 🐛 | After 🦋 |
| --- | --- |
|  |  |

# 📋 QA

- On the login screen, fill in the required fields
- Click on the ``` Log in ``` button

You should the new event registered in segment console

Screenshot 2021-03-13 at 07 22 40

# Story 📖

https://kickstarter.atlassian.net/browse/EP-276

### CODE REVIEW

File: app/src/main/java/com/kickstarter/viewmodels/LoginViewModel.kt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package com.kickstarter.viewmodels
 
+import android.util.Log
```
Comment (Arkariang) [inline app/src/main/java/com/kickstarter/viewmodels/LoginViewModel.kt:3]: Is this import necessary?

**Review by hadia — APPROVED** [review]
👍

File: app/src/test/java/com/kickstarter/viewmodels/LoginViewModelTest.kt [hunk 2]
```diff
@@ -14,6 +15,7 @@ import org.junit.Test
 import rx.Observable
 import rx.observers.TestSubscriber
 
+
```
Comment (leighdouglas) [inline app/src/test/java/com/kickstarter/viewmodels/LoginViewModelTest.kt:18]: nit: Remove whitespace

**Review by leighdouglas — APPROVED** [review]
Looks great, left a nit comment but otherwise good to go! 🎉

**Pull request merged**